static/app/components/sidebar/sidebarDropdown/index.tsx

```diff
@@ -93,24 +106,31 @@ const SidebarDropdown = ({api, org, orientation, collapsed, config, user}: Props
               {hasOrganization && (
                 <Fragment>
                   <SidebarOrgSummary organization={org} />
-                  {hasOrgRead && (
-                    <SidebarMenuItem to={`/settings/${org.slug}/`}>
-                      {t('Organization settings')}
-                    </SidebarMenuItem>
-                  )}
-                  {hasMemberRead && (
-                    <SidebarMenuItem to={`/settings/${org.slug}/members/`}>
-                      {t('Members')}
-                    </SidebarMenuItem>
-                  )}
+                  {!hideOrgLinks && (
```

Comment:
@billyvg I managed to simplify this component to only needing to use `hideOrgLinks` in one place, probably doesn't make sense to make it a different component when this is the only change